### PR TITLE
Remove the default column in the plugin doc page to give space to the sidebar

### PR DESCRIPTION
**Pull request by ph**
Fix: #6515

### CODE REVIEW

File: tools/logstash-docgen/templates/plugin-doc.asciidoc.erb [hunk 1]
```diff
@@ -46,7 +46,7 @@ Available configuration options:
 
 [cols="<,<,<,<m",options="header",]
```
Comment (suyograo) [inline tools/logstash-docgen/templates/plugin-doc.asciidoc.erb]: Needs to be removed here as well

Comment (ph) [inline tools/logstash-docgen/templates/plugin-doc.asciidoc.erb]: The whole line?

Comment (dedemorton) [inline tools/logstash-docgen/templates/plugin-doc.asciidoc.erb]: @ph You should use:

`[cols="<,<,<",options="header",]`

Comment (ph) [inline tools/logstash-docgen/templates/plugin-doc.asciidoc.erb]: thanks, this was non obvious to me.

**Comment by ph** [comment]
updated with @dedemorton suggestions.

**Comment by suyograo** [comment]
LGTM

**Pull request merged**